examples/ssm-examples.h

```diff
@@ -118,8 +118,11 @@ int main(int argc, char *argv[]) {
   }
 
   ssm_init_args = argv;
-
   ssm_time_t stop_at = stop_at_s == 0 ? SSM_NEVER : stop_at_s * SSM_SECOND;
+  printf("%s: simulating up to %lu seconds\n", prog,
```

Comment:
I added this print statement here to spook `printf` into "leaking" memory earlier; under the hood, it actually keeps around a 1024B buffer that it doesn't free until the very end, so we make sure it gets counted by the leak checkpoint take during `ssm_mem_init()`.

Reply:
Are you saying that printf will allocate a buffer and not free it until the program terminates? Since you are calling printf here, it will allocate a buffer, and then before it has a chance to free its buffer, you will check for memory leaks, which will include printf's buffer. Is this correct?